lib/localeapp/cli/pull.rb

```diff
@@ -2,10 +2,13 @@ module Localeapp
   module CLI
     class Pull < Command
       include ::Localeapp::ApiCall
+      @locale_key = nil
 
-      def execute
+      def execute(locale_key = nil)
         @output.puts "Localeapp Pull"
+        @output.puts "Fetching #{locale_key}" if locale_key
         @output.puts ""
+        @locale_key = locale_key
 
         @output.puts "Fetching translations:"
```

Comment:
See my comment about merging messages in the UAT scenario.

Then I would favor assigning instance variable first, then writing to
standard output, which I think we could refactor with only one `#puts`
call.